### ISSUE TITLE
Hide/reveal inside hide/reveal

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#466]
### Steps to Reproduce

1. Visit https://ocwnext.odl.mit.edu/courses/5-95j-teaching-college-level-science-and-engineering-fall-2015/
2. Click the hide/reveal widget (downward arrow) next to the course numbers

### Expected Behavior

Should be able to see all the course numbers 

### Actual Behavior

The course numbers are revealed, but the bottom of the list is still hidden within the course info containter

**Comment by MAbdurrehman12** [comment]
@pdpinch I don't think it is a good idea, in terms of UX, to have this nested expand/collapse widgets. User will have to click multiple arrows to view the content where as it can be done with only 1, the outer one. So I think we should work on to hide the inner expand/collapse widgets in `course info` for `home page` only but they should show when the `course info` is in the right side, on other pages.
What do you suggest?

**Comment by pdpinch** [comment]
@MAbdurrehman12 I agree, but I'm not sure what we will do about this yet. That's why I didn't put this issue on any of the boards yet.
